### PR TITLE
RUM-926: Update the limit for Android mapping file uploads to 300 MB

### PR DESCRIPTION
**Pull request by 0xnm**
### What does this PR do?
This PR updates Android mapping file limit from 200 MB to 300 MB for all Datadog sites except AP1 and GovCloud.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by hestonhoffman — APPROVED** [review]
LGTM. @0xnm am I good to merge?

**Comment by 0xnm** [comment]
@hestonhoffman we are waiting for the confirmation of the deployment of these changes, I will post it once confirmed.

File: content/en/real_user_monitoring/error_tracking/android.md
Comment (xgouchet) [inline content/en/real_user_monitoring/error_tracking/android.md:156]:
```suggestion
{{< site-region region="us,us3,us5,eu" >}}
Mapping files are limited to 300 MB. If your project has a mapping file larger than this, use one of the following options to reduce the file size:
{{< /site-region >}}
{{< site-region region="ap1,gov" >}}
Mapping files are limited to 50 MB. If your project has a mapping file larger than this, use one of the following options to reduce the file size:
{{< /site-region >}}
```

File: content/en/real_user_monitoring/error_tracking/reactnative.md
Comment (xgouchet) [inline content/en/real_user_monitoring/error_tracking/reactnative.md:53]:
```suggestion
{{< site-region region="us,us3,us5,eu" >}}
Datadog can accept uploads up to 300 MB.
{{< /site-region >}}
{{< site-region region="ap1,gov" >}}
Datadog can accept uploads up to 50 MB.
{{< /site-region >}}
```

**Review by xgouchet — APPROVED** [review]
Nit pick, we can use documentation's DC selector to change the text for each one

**Comment by 0xnm** [comment]
@hestonhoffman This PR is ready to be merged.

**Comment by xgouchet** [comment]
@hestonhoffman one thing that might be nice is to have a more consistent display of the limitation. For Android, it's part of a paragraph with no emphasis, for React Native, it's in a warning orange box, and for Javascript, it's part of the `Instrument your code` (and not _Limitations_), but with the sizes bold

**Comment by hestonhoffman** [comment]
@xgouchet Okay, let me take a look

**Comment by hestonhoffman** [comment]
@xgouchet I've pushed up an edit. The react and android pages no longer have alerts. The information is already listed under its own heading, so no alert is needed. In the guide, each section is now an alert because it suits the style of that document better. The numbers are now in bold everywhere.

I'm going to go ahead and merge this one in once all the checks pass

**Pull request merged**
